README.md

```diff
@@ -1 +1,58 @@
-# mvblogapp
\ No newline at end of file
+# Blog App
+
+## Getting Started
+
+To get a local copy of the project, follow these steps: 
+1. Open the terminal and navigate to the folder where you would like to put these files.
+2. Type the following commands into the terminal: 
+ ```
+ git clone https://github.com/Ol-create/MVBlogApp
+ ```
+ ```
+ cd blog-app
+ ```
+Check the ruby version being used and install it if it is not the same with that of the local machine
+Set up database direct them to the config/database.yml file
+```
+bundle install
+```
+```
+'npm install' or 'yarn install'
+```
+```
+rails db:create
+```
+```
+rails db:migrate
+```
+```
+rails 'db:seed' if there is data in the seed
+```
+```
+'rails s' to start the server locally
+```
+
+To execute the tests, run the following command in your terminal:
+```
+rspec
+```
+ 
+ ## Author
+
+👤 **Oluyemi Paul**
+
+- [GitHub: Ol-create](https://github.com/Ol-create)
+- [Twitter: OluyemiPaul99](https://twitter.com/OluyemiPaul99)
+- [LinkedIn: paul-oluyemi-193966ab](https://www.linkedin.com/in/paul-oluyemi)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] As asked by the previous reviewers, please use Microverse  [template](https://github.com/microverseinc/readme-template) for your readme, it is a requirement